### DISCUSSION TITLE
Feedback on Add Repository Links

### DISCUSSION THREAD

**Discussion by elmalakomar** [mia-platform/community#272] [Product Feedback]
### Which section?

Console

### Which is the version?

11.2.1

### Description

Hi,
in this version of the console it is available the new feature to manage the HTTP and SSH repository links to a microservice.

### Actual Outcome

If no repository link is associated to the microservice you see this button that allow you to define the HTTPS and SSH repository urls
![image](https://github.com/mia-platform/community/assets/18056946/a0b81f17-4e94-4aa8-9fcb-8e5f0e2ca55f)

IMHO, when the links are already set the UX is not so great:
![image](https://github.com/mia-platform/community/assets/18056946/a90cb50d-5ae0-4cc3-b10f-5819aa4b4c47)
as seen in the above picture, to manage the links you need to click the Clone button, but you don't want to clone the repos :smile: 
![image](https://github.com/mia-platform/community/assets/18056946/a1952425-a498-401c-b21b-84d981cbfdae)

### Expected Outcome

It could be more clear to have a third button like the "Add Repository Links" called "Edit Repository Links"

**Comment by albertotessarotto** [comment]
Hi! Thanks for the feedback. Could be "Manage repository links" or something more general compared with "edit" term? Seeing merely "edit" hides too much the related clone actions in my opinion.

**Reply by elmalakomar** [reply]
Yeah I guess your suggestion could be an option!

**Comment by nicola88** [comment]
@albertotessarotto any update on this or we can close it?
